### ISSUE TITLE
combat strength indicator priority

### ISSUE THREAD

**Issue by dotdotdot-tts** [rogeriocamorim/duneBugSuggestionTracker#26]
not sure how it's determined which color is on top, but i think the combat strength for a player over 20 should be on top of the player with less than 20.

blue @ 26 proof:
![blue over 20 proof](https://github.com/rogeriocamorim/duneBugSuggestionTracker/assets/154347378/32e7ea9e-46b0-4c9b-82ee-2ef6c91fedcd)

blue @ 26 under yellow @ 6:
![blue over 20 - but under yellow](https://github.com/rogeriocamorim/duneBugSuggestionTracker/assets/154347378/0bf383a3-50a9-459a-8cb1-eda29198dfc5)

**Comment by Chatanga** [comment]
It is based on the color actually. But why would it be better to sort them by value in the same stack? Isn't the amount of information the same in both cases?

**Comment by dotdotdot-tts** [comment]
it's a clarity thing.

if my 26 strength indicator disappears behind someone who has 6 strength and someone doesn't notice and makes a mental mistake due to it, it feels bad. 

there are many things to think through in uprising and while yes you can count the troops and worms, the combat strength number indicator is there so you can know where you and your opponents are at a glance. 

in such situations in the previous mod, players would manually put the person over 20 strength on top of the one under 20 strength for clarity, we cannot do that in this mod.

**Comment by aaittouares** [comment]
While i agree it could be better by value, it is zero priority for us now.
Also what about if 2 people got 20+ at same value ( for example both players are at 26) who has priority ?? 

Like always, TTS mods are not video games and players should always be responsible and look out on the table to see potential mistakes or things they forgot, like around a real table, we are only making scripts for QoL and not for being a " virtual rules enforcer". 

This enhancement will come later with No ETA.

**Comment by dotdotdot-tts** [comment]
i agree with that sentiment, but since we cannot modify the combat strength indicator without the scripted functionality, there is nothing else we can do but request better clarity haha. if we could manually put whatever we want on top, like we used to, this would be a moot point.
